scripts/push_docs_to_branch.sh

```diff
@@ -15,7 +15,7 @@ set -Eeuo pipefail
 TODAY=`date "+%Y%m%d"`
 # we build new versions only for minors and majors
 PATTERN_FOR_NEW_VERSION="^refs/tags/[0-9]+\\.0\\.0$"
-PATTERN_FOR_MICRO_VERSION="^refs/tags/[0-9]+\\.[0-9]+\\.[1-9][0-9]*$"
+PATTERN_FOR_EXISTING_VERSION="^refs/tags/[0-9]+\\.[0-9]+\\.[0-9]+$"
```

Comment:
also needs a change line 38 🤔

Reply:
Are you sure? 

for example 

```
GITHUB_REF=/refs/tags/3.0.1
```
So in such a case, we want to update the docs for `3.x`, right?

it will match with `elif [[ "$GITHUB_REF" =~ $PATTERN_FOR_MICRO_VERSION ]]` and `EXISTING_VERSION=$(echo $GITHUB_REF | sed -E "s/^refs\/tags\/([0-9]+)\.([0-9]+)\.[0-9]+$/\1.x/")` will return `3.x`

looks ok to me, or it's Monday morning and I miss something? 😄

Reply:
I think it's monday morning + not enough precise comment on my end. I meant renaming `PATTERN_FOR_MICRO_VERSION` to `PATTERN_FOR_EXISTING_VERSION` at line 38 😄

Reply:
I see 🙈